core/trino-spi/src/main/java/io/trino/spi/type/LongTimeWithTimeZoneType.java

```diff
@@ -36,7 +36,7 @@
 import static java.lang.String.format;
 import static java.lang.invoke.MethodHandles.lookup;
 
-class LongTimeWithTimeZoneType
+non-sealed class LongTimeWithTimeZoneType
```

Comment:
Why non-sealed? These should be final, instead.

Reply:
I have no idea... IntelliJ auto converted :(